requirements.txt

```diff
@@ -0,0 +1 @@
+-r requirements/heroku.txt
```

Comment:
Yes, we could do this. However we'd need to merge all requirements files in a single wagtaildemo/requirements.txt (Heroku needs the requirements.txt to be on the project's root directory).